coalib/testing/LocalBearTestHelper.py

```diff
@@ -117,15 +128,18 @@ def check_results(self,
                               list,
                               msg='The given results are not a list.')
 
+        if not msg:
+            msg = ("The local bear '{}' doesn't yield the right results."
+                   .format(local_bear.__class__.__name__))
+            if check_order:
+                msg += ' Or the order may be wrong.'
```

Comment:
maybe you should change the message to `The local bear '{}' yields no result although it should` here, if `results == []`? That moves the complexity to this function and avoids the `msg` parameter^^